docs/spec.md

```diff
@@ -495,6 +498,26 @@ Unsubscribe from [connection graph updates](#connection-graph-update). Only supp
 }
 ```
 
+### Fetch asset
+
+Fetch an asset from the server. Only supported if the server previously declared that it has the `assets` [capability](#server-info).
+
+#### Fields
+
+- `op`: string `"fetchAsset"`
+- `assetUri`: string, uniform resource identifier to locate a single asset
+- `requestId`: number, unique 32-bit unsigned integer which is to be included in the [response](#fetch-asset-response)
+
+#### Example
+
+```json
+{
+  "op": "fetchAsset",
+  "assetUri": "package://foo/robot.urdf",
```

Comment:
Maybe this should just be called `uri`? The op is already referring to an asset.